src/thumbnailer_near_lossless.cc

```diff
@@ -14,13 +14,12 @@
 
 #include "thumbnailer.h"
 
+namespace libwebp {
 // List of pre-processing values used in binary search for near-lossless to
 // speed-up the algorithm. It is not necessary to binary search for all the
 // values in range [0,100] since actually for near-lossless, the frame size and
 // PSNR are not changed when the preprocessing increases a small quantity.
-const int preprocessing_list[6] = {0, 20, 40, 60, 80, 100};
-
-namespace libwebp {
+const int kPreprocessingList[6] = {0, 20, 40, 60, 80, 100};
```

Comment:
it's better practice to also put this in an anonymous namespace, or you can add the "static" keyword before "const",  which has a similar effect